manifests/client/install.pp

```diff
@@ -2,13 +2,14 @@
 class mysql::client::install {
 
   if $mysql::client::package_manage {
-
-    package { 'mysql_client':
-      ensure          => $mysql::client::package_ensure,
-      install_options => $mysql::client::install_options,
-      name            => $mysql::client::package_name,
-    }
-
+		if $::osfamily == 'Windows' {  
+			include mysql::server::install
+		} else {
+			package { 'mysql_client':
+				ensure =>          $mysql::client::package_ensure,
+        install_options => $mysql::client::install_options,
+				name   =>          $mysql::client::package_name,
+			}  
+		}
```

Comment:
all of your indentation is broken here… please refer to the [styleguide](https://docs.puppetlabs.com/guides/style_guide.html)